OpenQA/Qemu/Proc.pm

```diff
@@ -430,6 +434,8 @@ sub stop_qemu {
 
 sub qemu_pid { shift->_process->process_id }
 
+sub dmesg { qx{dmesg} }
```

Comment:
hm, I hoped that the sub definition would be coverage enough :) Can you add

```suggestion
sub dmesg { qx{dmesg} }  # uncoverable statement
```

Please check if that is enough to ensure a complete coverage of your changes

Reply:
Added